Mathlib/Algebra/Polynomial/BigOperators.lean

```diff
@@ -193,10 +193,21 @@ theorem natDegree_multiset_prod_of_monic (h : ∀ f ∈ t, Monic f) :
     assumption
   · simp
 
+theorem degree_multiset_prod_of_monic [Nontrivial R] (h : ∀ f ∈ t, Monic f) :
```

Comment:
This is also true when R is trivial, right? Hopefully `nontriviality` makes it easy to strengthen this proof

Reply:
Sadly only at the expense of assuming that `t` is not empty; otherwise the LHS is `degree (prod 0) = degree 1 = degree 0 = ⊥` and the RHS is `sum (Multiset.map degree 0)) = sum 0 = 0`.